state-sync/storage-service/server/src/handler.rs

```diff
@@ -420,6 +409,9 @@ impl<T: StorageReaderInterface> Handler<T> {
             DataRequest::GetTransactionsOrOutputsWithProof(request) => {
                 self.get_transactions_or_outputs_with_proof(request)
             },
+            DataRequest::GetTransactionDataWithProof(request) => {
+                self.get_transaction_data_with_proof(request)
+            },
```

Comment:
We added 2 more variants in DataRequest -- For the optimistic variants, and for subscriptions. We need to add those too here?

Reply:
Yes! Added the code below 😄

Reply:
This match statement will still return Err for `DataRequest::GetNewTransactionDataWithProof` variant right?